packages/changed-elements-react/src/clients/iModelsClient.ts

```diff
@@ -8,8 +8,18 @@ export interface IModelsClient {
   /** Retrieves a list of Changesets, ordered by ascending `index` property.*/
   getChangesets(args: GetChangesetsParams): Promise<Changeset[]>;
 
-  /** Retrieves a list of Named Verions, ordered by ascending `changesetIndex` property. */
+  /** Retrieves a changeset*/
+  getChangeset(args: GetChangesetParams): Promise<Changeset | undefined>;
+
+  /** Retrieves a list of all Named Versions, ordered by ascending `changesetIndex` property. */
   getNamedVersions(args: GetNamedVersionsParams): Promise<NamedVersion[]>;
+
+  /** Retrieves a paged list of Named Versions */
+  getNamedVersionsPaged(args: GetNamedVersionsPagedParams): Promise<NamedVersion[]>;
+}
+
+export interface GetChangesetParams extends GetChangesetsParams {
```

Comment:
Please keep inheritance hierarchy use to the minimum. Extend just `CommonRequestParams` and add `iModelId` property explicitly.